src/mockServer/msw-browser.ts

```diff
@@ -0,0 +1,40 @@
+import { SetupWorker, setupWorker } from 'msw';
+
+import { Database, LocalContext } from '../types';
+import { buildMockLocalContext } from './fixtures';
+import { buildMSWMocks } from './msw-handlers';
+
+export const mockServiceWorkerServer = (
+  {
+    appContext,
+    database,
+  }: {
+    appContext: Partial<LocalContext> & Pick<LocalContext, 'itemId'>;
+    database?: Database;
+  },
+): { worker: SetupWorker; resetDB: (data: Database) => void } => {
+  const fullAppContext = buildMockLocalContext(appContext);
+  const mswMocks = buildMSWMocks(fullAppContext, database);
+  mswMocks.db.on('populate', (transaction) => {
+    if (database) {
+      // seed database with data
+      // eslint-disable-next-line no-console
+      console.log('Populating the DB with provided mock data');
+      transaction.table('item').bulkAdd(database?.items);
+      transaction.table('member').bulkAdd(database?.members);
+      transaction.table('appData').bulkAdd(database?.appData);
+      transaction.table('appAction').bulkAdd(database?.appActions);
+      transaction.table('appSetting').bulkAdd(database?.appSettings);
+      transaction.table('appContext').bulkAdd([database?.appContext]);
```

Comment:
`bulkAdd` works with `undefined`?

Reply:
No it complains, also it complains when the array is empty. I added checks.